Gems/ScriptCanvas/Assets/TranslationAssets/Classes/PaintBrushSettings.names

```diff
@@ -0,0 +1,366 @@
+{
+    "entries": [
+        {
+            "base": "PaintBrushSettings",
+            "context": "BehaviorClass",
+            "variant": "",
+            "details": {
+                "name": "Paint Brush Settings"
+            },
+            "methods": [
+                {
+                    "base": "GetsmoothingRadius",
```

Comment:
Is the casing of `GetsmoothingRadius` and below methods intentional?

Reply:
The base values are just set to whatever the "generate translation" in script canvas produces into the file. I manually edited the "name" values (which are the display values) to look correct. I have no idea what the base values are used for, I assume they're just an internal key and don't really matter too much what they look like.